### DISCUSSION TITLE
Trouble with Jupyter

### DISCUSSION THREAD

**Discussion by amm20052** [cooperrc/computational-mechanics#219] [Q&A]
I moved the Computational Mechanics folder to my work folder on Jupyter. I can see a Computational Mechanics folder in my work folder, but there is also one that still exists on the main page. Additionally, when I create new work from the course website (when I click on the rocket ship to open the content in Jupyter) it puts the new work inside the original Computational Mechanics folder, not the one that I moved to my work folder. I am worried that this means it is not saving properly. As far as I can tell there is no way to manually move my work to the other folder. How do I fix this?

**Comment by BendlakJustin** [comment]
I believe that as long as you keep moving it into your work folder every time you create new work, you should be fine.

**Comment by cooperrc** [comment]
That's true, each time you click the rocket icon, it will create a file in the home folder _not the work folder_.

You can use File -> save as -> type in work/computational-mechanics/module_01/HW_01.md and choose "overwrite" 

_or_

open the files in the Jupyter file browser each time
